README.md

```diff
@@ -1,53 +1,220 @@
-# Getting Started with the OVM: Simple ERC20 Token Truffle Project
+# Getting Started with the Optimistic Ethereum: Simple ERC20 Token Truffle Tutorial
 
-### WARNING: THIS TUTORIAL IS OUTDATED. PLEASE SEE https://github.com/ethereum-optimism/optimism-tutorial FOR AN UP TO DATE TUTORIAL
+Hi there! Welcome to our Optimistic Ethereum ERC20 Truffle example!
 
-Hi there! Welcome to our OVM ERC20 Truffle tutorial.
+If your preferred smart contract testing framework is Waffle, see our Optimistic Ethereum ERC20 Waffle tutorial [here](https://github.com/ethereum-optimism/Waffle-ERC20-Example).
 
-If your preferred smart contract testing framework is Waffle, see our OVM ERC20 Waffle tutorial [here](https://github.com/ethereum-optimism/Waffle-ERC20-Example). If you're interested in writing your first L2-compatible smart contract using Truffle as your smart contract testing framework, then you've come to the right place!  This repo serves as an example for how to test running your contracts in the EVM and OVM and how to uncover OVM bugs for the Optimism team to fix.
+If you're interested in writing your first L2-compatible smart contract using Truffle as your smart contract testing framework, then you've come to the right place!
+This repo serves as an example for how go through and compile/test/deploy your contracts on both Ethereum and Optimistic Ethereum.
 
-### Prerequisites
+Let's begin!
+
+## Prerequisites
+
+Please make sure you've installed the following before continuing:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Node.js](https://nodejs.org/en/download/)
+- [Yarn 1](https://classic.yarnpkg.com/en/docs/install#mac-stable)
+- [Docker](https://docs.docker.com/engine/install/)
+
+## Setup
+
+To start, clone this `Truffle-ERC20-Example` repo, enter it, and install all of its dependencies:
+
+```sh
+git clone https://github.com/ethereum-optimism/Truffle-ERC20-Example.git
+cd Waffle-ERC20-Example
+yarn install
 ```
-node v11.10.1
+
+## Step 1: Compile your contracts for Optimistic Ethereum
+
+Compiling a contract for Optimistic Ethereum is pretty easy!
+First we'll need to install the [`@eth-optimism/solc`](https://www.npmjs.com/package/@eth-optimism/solc).
+Since we currently only support `solc` versions `0.5.16`, `0.6.12`, and `0.7.6` for Optimistic Ethereum contracts, we'll be using version `0.7.6` in this example.
+Let's add this package:
+
+```sh
+yarn add @eth-optimism/solc@0.7.6-alpha.1
 ```
 
-## Set up
+Next, we just need to add a new `truffle-config-ovm.js` file to compile our contracts.
 
-To start out, clone this example repo
+First, since in our new config file we'll be using a package called `@truffle/hdwallet-provider` to get a list of accounts with preexisting ether balances, we'll need to install this package:
 
-```bash
-git clone https://github.com/ethereum-optimism/Truffle-ERC20-Example.git
+```sh
+yarn add @truffle/hdwallet-provider
```

Comment:
Sorry for the flip-flop here: could we include `@truffle/hdwallet-provider` in the `package.json` instead of asking users to install it during the tutorial? Seems a bit easier. Generally I don't think it's necessary to explain the `HDWalletProvider`.